vmdb/app/assets/javascripts/miq_application.js

```diff
@@ -1011,26 +1034,34 @@ function miqAttachTextAreaWithLines(id) {
   el.scrollTop = ta.scrollTop;
 
   ta.focus();
-  ta.onkeydown = function() { el.scrollTop = ta.scrollTop; };
-  ta.onmousedown = function() { el.scrollTop = ta.scrollTop; };
-  ta.onmouseup = function() { el.scrollTop = ta.scrollTop; };
-  ta.onmousemove = function() { el.scrollTop = ta.scrollTop; };
```

Comment:
can we have just one function here and pass the function to all the events here?